src/components/20-molecules/cookie-disclaimer/index.js

```diff
@@ -32,6 +32,8 @@ class AXACookieDisclaimer extends LitElement {
     this.onClick = () => {};
   }
 
+  //  With this, it removes it itself if you use the static <axa-cookie-disclaimer> tag.
+  //  Needs to be re-evaluated.
```

Comment:
i dont think it needs to be reevaluated, can u remove this text please

Reply:
True. Removed.